version/version.go

```diff
@@ -14,7 +14,14 @@
 
 package version
 
-import "github.com/coreos/go-semver/semver"
+import (
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/Sirupsen/logrus"
```

Comment:
:-1: drone is using zerolog

Reply:
Is it imported there? Nope.

Reply:
Damn, on the server it's still used

Reply:
Yeah, it quite literally is: https://github.com/drone/drone/blob/master/cmd/drone-server/server.go#L46

Reply:
Also, speaking from experience, JSON logs are a _terrible_ idea.

Reply:
From our experience they are working pretty well.